src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -38,6 +38,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="SequenceReader\CopyTo.cs" />
```

Comment:
Sort alphabetically and move to the item group that contains the other SequenceReader tests (up above in this file).

Reply:
Sorry, I had this in my change stacked on top of this (adding readonly). It will be up shortly.

Reply:
Does that include the optimized check in Rewind:
https://github.com/dotnet/corefx/pull/37416#discussion_r280967434